editor/inspector_dock.cpp

```diff
@@ -584,6 +584,7 @@ InspectorDock::InspectorDock(EditorNode *p_editor, EditorData &p_editor_data) {
 	add_child(warning);
 	warning->set_text(TTR("Changes may be lost!"));
 	warning->set_icon(get_icon("NodeWarning", "EditorIcons"));
+	warning->set_clip_text(true);
```

Comment:
Out of curiosity, is there a reason this was set here? Or does this fix an unrelated problem?